associated-press-feed/cdk/lib/associated-press-feed.ts

```diff
@@ -0,0 +1,28 @@
+import { GuPlayApp } from '@guardian/cdk';
+import { AccessScope } from '@guardian/cdk/lib/constants';
+import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
+import { GuStack } from '@guardian/cdk/lib/constructs/core';
+import type { App } from 'aws-cdk-lib';
+import { InstanceClass, InstanceSize, InstanceType } from 'aws-cdk-lib/aws-ec2';
+
+export class AssociatedPressFeed extends GuStack {
+	constructor(scope: App, id: string, props: GuStackProps) {
+		super(scope, id, props);
+		new GuPlayApp(this, {
+			app: 'associated-press-feed',
+			access: { scope: AccessScope.INTERNAL, cidrRanges: [] },
+			instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MICRO),
+			monitoringConfiguration: { noMonitoring: true },
```

Comment:
For my own knowledge, would there be an advantage to adding monitoring here?
```suggestion
			monitoringConfiguration: {
				/*
				The alarm will trigger if: >= 2 runs fail within 7 minutes
				*/
				toleratedErrorPercentage: 99,
				numberOfMinutesAboveThresholdBeforeAlarm: 7,
				datapointsToAlarm: 2,
				snsTopicName: `TODO`,
			},
```